### ISSUE TITLE
copy-source to another space fails summary

### ISSUE THREAD

**Issue by Zteve** [cloudfoundry/cli#762]
The following command:
```
cf copy-source java-main-application jta2 -o integration-test -s integration-test
```
executed in a different org/space than `integration-test` and for running applications in both places, produces the following output:
```
Copying source from app java-main-application to target app jta2 in org integration-test / space integration-test as admin...
Note: this may take some time

Stopping app jta2 in org integration-test / space integration-test as admin...
OK

Starting app jta2 in org integration-test / space integration-test as admin...

0 of 1 instances running, 1 starting
0 of 1 instances running, 1 starting
1 of 1 instances running

App started
OK

App  was started using this command ``

Showing health and status for app  in org integration-test / space integration-test as admin...
FAILED
Server error, status code: 404, error code: 100004, message: The app could not be found: summary
```

Everything fine until `App  was started using this command ''` which contains a blank (empty string) application name.  Thereafter the name `summary` appears to be used and the command fails.

Tracing reveals that the application `jta2` is looked up in the current space (rather than the target space) and that no id is found (unsurprisingly), so a blank `app-guid` is passed to a `GET /v2/apps/:app-guid/summary` request. This results in the strange last line and the non-zero return code.

CLI version `cf version 6.15.0+fa1bfe2-2016-01-13` is being used.

**Comment by dkoper** [comment]
Thanks @Zteve! I was able to reproduce it, we'll investigate.

**Comment by dkoper** [comment]
This has been fixed in cf CLI v6.16.0.
https://github.com/cloudfoundry/cli#downloads

Regards,
Dies Koper
CF CLI PM
